sql/2024/performance/bfcache_cachecontrol_nostore.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TEMP FUNCTION HAS_NO_STORE_DIRECTIVE(cache_control STRING) RETURNS BOOL AS (
+  REGEXP_CONTAINS(cache_control, r'(?i)\bno-store\b')
+);
+
+WITH requests AS (
+  SELECT
+    client,
+    LOGICAL_OR(HAS_NO_STORE_DIRECTIVE(JSON_VALUE(payload, '$._cacheControl'))) AS includes_ccns
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-06-01' AND
+    is_main_document
+  GROUP BY
+    client,
+    page
+)
```

Comment:
Should we group by `root_page` rather than page? That way you can report by site rather than per page (given HTTP Archive is limited to two pages per site).

Reply:
Updated to use `root_page`. Updated results: https://docs.google.com/spreadsheets/d/15038wEIoqY53Y_kR8U6QWM-PBO31ZySQGi147ABTNBc/

Reply:
Change in plans. Will use `page` and update the unload handler to use `page`. This is more aligned with the previous version (2022) and the writing.